content/2_Bronze/Intro_DS.mdx

```diff
@@ -159,8 +159,80 @@ dynamic arrays (e.g. `vector<vector<int>>`) static arrays of dynamic arrays
 (e.g. `array<vector<int>,5>`), dynamic arrays of static arrays (e.g.
 `vector<array<int,5>>`), and so on.
 
-<!-- Usually, the choice between a static array and a dynamic array is just personal preference. -->
+## Doubly-Linked Lists
```

Comment:
are there any problems in silver or below where this would be useful? if not maybe we should leave it out (and just link to the `std::list` documentation)

Reply:
There are old bronze problems which can be solved (I don't have the link but remember doing one) which used it.

Reply:
er, I don't think we should add this w/o an example

Reply:
why resolved?

Reply:
https://www.cs.hmc.edu/ACM/Problems/Fall12/mooo.txt

This one can be solved with Doubly Linked Lists. I don't think there is a POJ